keychain/btcwallet.go

```diff
@@ -472,10 +472,15 @@ func (b *BtcWalletKeyRing) SignMessageSchnorr(keyLoc KeyLocator,
 		privKey = txscript.TweakTaprootPrivKey(*privKey, taprootTweak)
 	}
 
+	// If a tag was provided, we need to take the tagged hash of the input.
 	var digest []byte
-	if doubleHash {
+	switch {
+	case len(tag) > 0:
+		var taggedHash = chainhash.TaggedHash(tag, msg)
```

Comment:
nit: can just do `taggedHash := chainHash.TaggedHash(tag, msg)` instead of using a `var` here.

Reply:
`var` is mostly used to define a variable that hasnt been assigned yet _or_ to group a bunch of `:=` statements together. 

Just a style nit